spec/config.yaml

```diff
@@ -494,12 +494,12 @@ mappings:
       starts: {type: integer}
       strands: {type: keyword}
       stops: {type: integer}
-      functions: {type: text}
-      functional_descriptions: {type: text}
-      warnings: {type: text}
+      functions: {type: keyword}
```

Comment:
Are these being changed back to what they were. I changed them earlier to match up with the previous versions of the indexers

Reply:
They are being changed to the type mappings we have deployed on CI. Setting all this to type:text produces a bunch of type mapping conflict errors. 

It seems like at some point things got a little confused. It seems to me like the types in this PR are the original type mappings deployed to CI and prod